internal/config/config.go

```diff
@@ -434,3 +436,14 @@ func ValidateVisibility(visibility string) bool {
 
 	return false
 }
+
+// ValidateRetrySettings checks the retries, maxAttempt and minPass settings
+func ValidateRetrySettings(cfg SauceConfig) string {
```

Comment:
Seems a bit odd to me for a function named `Validation*` to return a string instead of a bool. Also makes the unit test brittle since the tests will fail if the error messages change.